src/entities/activity/ui/items/ActionPlan/Header.tsx

```diff
@@ -0,0 +1,19 @@
+import { useXScaledDimension } from './hooks';
+
+import { Theme } from '~/shared/constants';
+import Text from '~/shared/ui/Text';
+
+export const Header = ({ children }: { children: string }) => {
+  const headerFontSize = useXScaledDimension(20);
+
+  return (
+    <Text
+      fontSize={`${headerFontSize}px`}
+      fontWeight="400"
+      lineHeight="28px"
+      color={Theme.colors.light.onSurfaceVariant}
```

Comment:
Some stying discrepancies:
```suggestion
      fontWeight="700"
      lineHeight="28px"
      color={Theme.colors.light.onSurface}
```